src/testing/cluster/storage_checker.zig

```diff
@@ -73,23 +69,34 @@ pub const StorageChecker = struct {
         var free_set = try vsr.FreeSet.init(allocator, Storage.grid_blocks_max);
         errdefer free_set.deinit(allocator);
 
+        var client_sessions = try vsr.ClientSessions.init(allocator);
+        errdefer client_sessions.deinit(allocator);
+
         var free_set_buffer = try allocator.alignedAlloc(
             u8,
             @alignOf(u64),
             vsr.FreeSet.encode_size_max(Storage.grid_blocks_max),
         );
-        errdefer allocator.free(free_set);
+        errdefer allocator.free(free_set_buffer);
```

Comment:
Oups! Good catch! I've been thinking recently that maybe we should flat out `@panic` on OOM
errors, so that we could get rid of `errdefer`s like these...